src/Build/Logging/BinaryLogger/BuildEventArgsReader.cs

```diff
@@ -806,14 +806,16 @@ private BuildEventArgs ReadTaskStartedEventArgs()
             var taskName = ReadOptionalString();
             var projectFile = ReadOptionalString();
             var taskFile = ReadOptionalString();
+            var taskAssemblyNameString = ReadOptionalString();
```

Comment:
```suggestion
            var taskAssemblyNameString = _fileFormatVersion > 19 ? ReadOptionalString() : null;
```

Let's make sure this can read older binlogs as well.

Reply:
Good one, fixed.